packages/core-snapshots-cli/lib/db/queries/transactions/delete-from-timestamp.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM TRANSACTIONS
```

Comment:
Properties in queries should be all lower-case, only SQL keywords should be uppercase.